test/fixtures/output/kafka/topic/update-success-rest-partitions-count.golden

```diff
@@ -0,0 +1,4 @@
+Updated the following configuration values for topic "topic-exist-rest" (read-only configs were not updated):
```

Comment:
Any way we can omit the message in parenthesis if there are no read-only configs?

Reply:
I can use two different msg's?

Reply:
Or just append the part in the parenthesis to the message if you detect one or more read-only configuration values.

Reply:
This is a good change. I've definitely seen at least one user that got confused by this message showing up when they didn't try to change read-only configs.